whatsapp_archive.py

```diff
@@ -271,32 +271,18 @@ def FormatHTML(data):
             <ol class="messages">
             {% for _, _, body, media in messages %}
                 {% if media is not none %}
-                    {% if "IMG" in media %}
-                        <li>
+                    {% if "IMG" in media or "jpeg" in media or "png" in media or "tiff" in media or "ico" in media or "gif" in media %}
```

Comment:
This doesn't seem to scale well. It should probably be a list in the template data, and here there should be just one expression. Something like `{% if media.IsImage() %}` or `{% if media.type in image_types %}` or something readable like that.

Also, what if there's a file called "jpeg.gif", will your code work correctly?